HW_3/src/main/java/com/epam/homework3/repository/impl/BookingRepositoryImpl.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.homework3.repository.impl;
+
+import com.epam.homework3.model.entity.Booking;
+import com.epam.homework3.repository.BookingRepository;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component

```

Comment:
Use @Repository annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers